lib/lang/isEmpty.js

```diff
@@ -7,23 +7,23 @@ var hasValues = require('has-values');
  * functions, numbers, strings, nulls, objects and arrays.
  *
  * ```js
- * utils.lang.isEmpty('a');
- * //=> true
- *
- * utils.lang.isEmpty('');
+ * isEmpty('a');
  * //=> false
  *
- * utils.lang.isEmpty(1);
+ * isEmpty('');
```

Comment:
another thought, if we don't add `utils.foo` before functions, then I should make it more clear what category a function is in on the docs, so it's easier to see when scrolling